articles/search/search-get-started-vector.md

```diff
@@ -259,18 +265,17 @@ api-key: {{admin-api-key}}
         ],
         "fields": "contentVector",
         "k": 5
-    },
-    "select": "title, content, category"
+    }
 }
 ```
 
 The response includes 5 results, and each result provides a search score, title, content, and category. In a similarity search, the response will always include "k" results ordered by the value similarity score.
 
 ### Single vector search with filter
 
-You can add filters, but the filters are applied to the nonvector content in your index. In this example, the filter applies to the "category" field.
+You can add filters to filter on metadata fields. In this example, the filter applies to the "category" field.
```

Comment:
category isn't a metadata field.... in the sense that we use the term "metadata fields" for things like blob properties, or table properties, etc.  what was wrong or misleading about "nonvector content"?

Reply:
Fair, reverted.